src/jojoe77777/FormAPI/CustomForm.php

```diff
@@ -83,6 +105,9 @@ public function addSlider(string $text, int $min, int $max, int $step = -1, int
         }
         $this->addContent($content);
         $this->labelMap[] = $label ?? count($this->labelMap);
+        $this->validationMethods[] = static function($v) use($min, $max) : bool {
+            return is_float($v) && $v >= $min && $v <= $max;
```

Comment:
`is_float($v)` is equivalent to `gettype($v) === 'float'`. It'll return false on integers.

Reply:
Can the client response with an integer for sliders? In the few tests I done it always responded with floats..

Reply:
If it's a whole number, it might, and even if it didn't, it should be allowed since user code will be able to handle it anyway.

Reply:
What do you suggest I do? Something like ``(is_float($v) || is_int($v))`` ?

Reply:
Yes. If I remember correctly, the JSON spec does not distinguish floats from ints, so it is perfectly correct to encode a float `1.0` as "1" in JSON.